apps/golden-sample-app-e2e/project.json

```diff
@@ -10,6 +10,44 @@
       "options": {
         "command": "npx playwright test --project 'mobile-chrome' --grep '@visual.+@responsive|@responsive.+@visual' --workers=2"
       }
+    },
+    "e2e-localhost-ebp-sndbx": {
+      "executor": "@dot-build/serve-and-run-angular:run",
+      "options": {
+        "devServerTarget": "golden-sample-app:serve:ebp-sndbox",
+        "command": "npx playwright",
+        "args": [
+          "test",
+          "--config=playwright.localhost.config.ts",
+          "--project=localhost-ebp-sndbx",
+          "--grep-invert '@visual'"
+        ]
+      }
+    },
+    "e2e-localhost-mocks": {
+      "executor": "@dot-build/serve-and-run-angular:run",
+      "options": {
+        "devServerTarget": "golden-sample-app:serve:mocks",
+        "command": "npx playwright",
+        "args": [
+          "test",
+          "--config=playwright.localhost.config.ts",
+          "--project=localhost-mocked",
+          "--grep-invert '@e2e'"
```

Comment:
Curious to know why this is excluding the `@e2e` tag. Should this be `@visual`? Same for `e2e-modelbank-stg` on [line 48](https://github.com/Backbase/golden-sample-app/pull/344/files#diff-ad0de0d5757c7e0982518bdde9ed2394645ab6bd1c5036445960d1d547783112R48).

Reply:
I think renaming the tags is fine. This is purely for example/demonstration purposes. So long as the tags make sense it's fine with me.

Reply:
Updated tags